Example/STKWebKitViewController.xcodeproj/project.pbxproj

```diff
@@ -502,6 +502,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "STKWebKitViewController/STKWebKitViewController-Prefix.pch";
 				INFOPLIST_FILE = "STKWebKitViewController/STKWebKitViewController-Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
Sounds good. Let me wait a few days and see if my pull gets accepted into the 1password extension. If it doesn't I will look into other options.